static/app/components/events/interfaces/frame/line.tsx

```diff
@@ -490,11 +471,6 @@ const StyledIconRefresh = styled(IconRefresh)`
   margin-right: ${space(0.25)};
 `;
 
-const LeadHint = styled('div')<{width?: string}>`
-  ${p => p.theme.overflowEllipsis}
-  max-width: ${p => (p.width ? p.width : '67px')}
```

Comment:
I'm pretty sure this ellipsis and max-width aren't necessary anymore with the recent stack trace style changes (at least, all the stack traces I've found still look as expected after removing this)